docs/contracts/introduction.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Introduction to Smart Contracts on Substrate"
+---
+
+Welcome to the smart contracts section of the Developer Hub!
+
+Here you can lean how to write WebAssembly ([Wasm](overview/glossary.md#webassembly-wasm)) smart contracts for Substrate block chains using the [Contract](https://github.com/paritytech/substrate/tree/master/srml/contract) module from the Substrate Module Runtime Library ([SRML](https://substrate.dev/docs/en/overview/glossary#srml-substrate-runtime-module-library)).
+
+Because Substrate supports Wasm smart contracts, it means that any language that can compile to Wasm could be used to write these contracts. ink! is Parity's answer for writing smart contracts and is an embedded domain specific language (eDSL) based on the Rust programming language.
+
+Existing smart contract platforms like [Ethereum](https://www.ethereum.org/) have heavily influenced the design of both the Contract module and the ink! programming language. The documentation in this section may make assumptions that you have previous knowledge of common concepts and patterns of these platforms.
+
+## Contract Module
+
+The SRML Contract module provides the ability for the runtime to deploy and execute WebAssembly smart contracts. Here we will provide a short overview of the common similarities and differences between the Contract module and Ethereum. To really learn all of the fine details, you can take a look at the [reference documentation for `srml_contract`](https://crates.parity.io/srml_contract/index.html).
+
+### Similarities to Ethereum
+
+As mentioned above, the Contract module shares many similarities to Ethereum. It uses an account based system where each smart contract instance has:
+
+* AccountId
+* Balance
+* Associated Code
+* Storage
+
+The [halting problem](https://en.wikipedia.org/wiki/Halting_problem) is solved by using a gas/fee system.
+
+The Contract module provides safety to the blockchain state by enabling revertable transactions, which rollback any changes to the storage by contract calls which fail.
+
+Contract calls can change alter the storage of the contract, create new contracts, and call other contracts.
+
+### Differences to Ethereum
+
+Unlike Ethereum, the Contracts module uses a WebAssembly interpreter ([wasmi](https://github.com/paritytech/wasmi/)) to execute smart contracts.
```

Comment:
The interpreter is just an implementation detail. The important difference that Ethereum uses EVM and srml-contract uses WebAssembly as the VM.